vehicle/src/main/scala/com/wedriveu/vehicle/control/VehicleBehavioursImpl.scala

```diff
@@ -27,55 +24,62 @@ trait VehicleBehaviours {
 
 }
 
-class VehicleBehavioursImpl(vehicle: SelfDrivingVehicle) extends VehicleBehaviours {
-   var notRecharging: Boolean = true
+class VehicleBehavioursImpl(selfDrivingVehicle: SelfDrivingVehicle) extends VehicleBehaviours {
    val zeroBattery: Double = 0.0
    val batteryThreshold: Double = 20.0
    val stateRecharging: String = "recharging"
-   val batteryToConsume: Double = 1.0
+   val batteryToConsume: Double = 0.14
    val newPositionIsTheSame: String = "The position given is the same, the vehicle doesn't move"
-   val conversionInMillis: Int = 3600 * 1000
    val timeOfJourney: Long = 0.0.asInstanceOf[Long]
-   val timeStep: Long = 1000.0.asInstanceOf[Long]
```

Comment:
Since these `val` are declares as `Long`, are you sure you need the `asInstanceOf` method? If so, no problem, but please check if its necessary or if you just need to remove the `.0` from for example the `1000.0` value